### DISCUSSION TITLE
Add request data to fingerprint

### DISCUSSION THREAD

**Discussion by mvenghaus** [magewirephp/magewire#89] [Ideas]
There are a lot of cases where you need to interact with the initial request data. But there is no access to the actual request uri or params. You can save them in your component using the boot method .. but doing it over and over is not really nice :)

So why not adding them to the fingerprint!?

![image](https://user-images.githubusercontent.com/6408081/229333678-d8f8f97d-e7d6-4109-9936-7a5beb28071d.png)

**Comment by Aquive** [comment]
Great idea!
